Client/Frontend/Settings/AdblockDebugMenuTableViewController.swift

```diff
@@ -46,8 +46,7 @@ class AdblockDebugMenuTableViewController: TableViewController {
             """
         
         var section = Section(header: .title("Available lists"), footer: .title(footerNote))
-        guard let store = WKContentRuleListStore.default() else { return completion }
-        
+        let store = ContentBlockerHelper.ruleStore
```

Comment:
I assume that accessing this is safe in Swift land, right? Trying to recall the rules for when static variables are initialized and if they're done atomically…

Reply:
no idea really, shouldn't it crash then with EXC_BAD_ACCESS?

I would say, keep it in mind and if the problem with threading and rulelist compilation persist, try to dig more and make it more robust